frontend/public/components/utils/cog.tsx

```diff
@@ -14,7 +14,7 @@ import { connectToModel } from '../../kinds';
 const CogItems: React.SFC<CogItemsProps> = ({options, onClick}) => {
```

Comment:
Given this component is no longer a cog, it seems like we should rename it?

Reply:
Wanted to make that a follow-on since it's a huge change: https://github.com/openshift/console/pull/432#issuecomment-428974812

Reply:
Ok.  I'm moving some CSS around, which is what prompted me to look.  We're gonna have a conflict, so we'll have to work to resolve depending on which PR merges first.